NewForce-Cohort-5/tabloidfullstack-spring-of-deception
# View all categories

# Description

Added: GetAllCategories function for the user to view all categories upon clicking the "categories" link in the header.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ X ] New feature (non-breaking change which adds functionality)

# Testing Instructions

git add and git commit all changes in your current branch.
git checkout main
git fetch -a
git checkout ViewAllCategories
Navigate to the root directory for this repo (directory-to)/tabloidfullstack-spring-of-deception
Start Tabloid.sln
Make sure your db is created with seed data (Skip to 9 if this is already done)
Run [01_Tabloid_Create_DB.sql](https://github.com/NewForce-Cohort-5/tabloidfullstack-spring-of-deception/blob/main/SQL/01_Tabloid_Create_DB.sql) and [02_Tabloid_Seed_Data.sql](https://github.com/NewForce-Cohort-5/tabloidfullstack-spring-of-deception/blob/main/SQL/02_Tabloid_Seed_Data.sql) queries in Visual Studio
Start the server in Debug Mode F5
Log in as admin [foo@bar.com](mailto:foo@bar.com) (or any user)
Navigate to the Category link
A categories page will appear showing the name of each category.


# Checklist:

- [ X ] My code follows the style guidelines of this project
- [ X ] I have performed a self-review of my own code
- [ X ] I have commented my code, particularly in hard-to-understand areas
- [ X  ] My changes generate no new warnings or errors
- [ X ] I have added test instructions that prove my fix is effective or that my feature works
